en/developer_guide/block_parameter.rst

```diff
@@ -0,0 +1,127 @@
+Block parameter
+===============
+
+Context
+-------
+
+For their display to be customizable, some blocks need to receive parameters when they render.
+Those parameters could be of multiple types, for instance :
+
+- url parameters
+- server parameters
+- header parameters
+- ...
+
+An simple solution to give those parameters to the block would have been to give all the request
+parameters when the block is called.
+
+This solution is not optimal for a main reason: when loading blocks via `ESI`_, each
+block will be cached for each url variant.
+
+If the block 1, which does not require any parameters, will be cached for each url variant :
+
+- ``/block/1`` will be in one cache entry.
+- ``/block/1?foo=bar`` will be in another cache entry.
+
+To improve your cache hit ratio, you should have the least entries for a given page.
+
+Open Orchestra solution
+-----------------------
+
+Open Orchestra lets you specify the parameters to give to a block in order to limit the cache
+entries for a single block.
+
+This solution requires work in two specifics areas :
+- in the Back Office, where you add the block
+- in the Front Office, where you will require some parameters linked to the block.
+
+Backoffice work
+~~~~~~~~~~~~~~~
+
+In the Backoffice, we provide a way to store the parameters linked to the block in the block reference
+of the areas.
```

Comment:
I think this should be rephrased to be clearer: 
"When a block is added in an area of a node, a block reference is created to link the block and the area. That block reference contains the parameter types required to display the block in Front Office.